Lib/typing.py

```diff
@@ -818,6 +818,9 @@ def __init__(self, name, *constraints, bound=None,
         if def_mod != 'typing':
             self.__module__ = def_mod
 
+    def __mro_entries__(self, bases):
```

Comment:
It should be added not here, but in `_TypeVarLike`. `ParamSpec` instances are not subclassable too.

Reply:
And add it also in TypeVarTuple.

Reply:
Thanks for the feedback! I'll get to this soon and ping you when done (I'm currently in the process of moving across the country and starting a job)